packages/netlify-cms-widget-media/src/ImagePreview.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
```

Comment:
Where is this file being used?

Reply:
nowhere as far as I'm aware. this was already included when I first cloned the project. I'm fairly new to react so I assumed it must've been there for a reason.

I am guessing it would be safe to remove this?

Reply:
In the original image widget this file is used to display the image in the CMS preview pane (right pane when you edit an entry).
The media widget should probably have the same functionality.